src/tasks/components/TaskRunsCard.tsx

```diff
@@ -66,7 +74,7 @@ class UnconnectedTaskRunsCard extends PureComponent<
           <ResourceCard.Meta>
             {this.activeToggle}
             <>Created at: {task.createdAt}</>
```

Comment:
let's take this opportunity to format this date

Reply:
great idea!